e2e_tests/tests/config.py

```diff
@@ -105,7 +105,7 @@ def set_aggregation_frequency(config: Dict[Any, Any], aggregation_frequency: int
     return config
 
 
-def set_amp_level(config: Dict[Any, Any], amp_level: str) -> Dict[Any, Any]:
+def set_apex_level(config: Dict[Any, Any], amp_level: str) -> Dict[Any, Any]:
```

Comment:
Can you just remove this function?  Since we removed the pre-pytorch-flexible-primitives API, this config setting is now totally unused

Reply:
I'm pretty sure at least 1 customer still has it in use. I think we should generally deprecate APEX anyway, but let's leave a window of compatibility until this has been used successfully in the wild and then deprecate / remove all of APEX.

Reply:
Ooh okay - someone told me that they really liked that Apex was enabled with config only and no code changes, so I assumed that's what they were actually doing.

Reply:
Oh I didn't realize this was in tests, too. So the tests have just not been exercising AMP. I'll actually add an APEX test alongside my auto / manual tests for now, with the expectation that we deprecate it as soon as native AMP is definitely ready for prime time.